src/main/java/seedu/bankwithus/Ui.java

```diff
@@ -6,6 +6,10 @@ public class Ui {
 
     private Scanner scanner;
 
+    public static void showSaveGoalCreated(String args, String untilWhenStr) {
+        System.out.println("try saving a Minimum of $"+args+" until "+untilWhenStr);
+        System.out.println("Save Goal has been created, Have fun staying frugal!");
+    }
```

Comment:
Please don't use static unless absolutely necessary, which in this case it is not. Using ui.showSaveGoalCreated rather than Ui.showSaveGoalCreated will fix the static call error